python/GafferImageTest/ImageWriterTest.py

```diff
@@ -368,7 +368,7 @@ def testMultipleWrite( self ) :
 		r = GafferImage.ImageReader()
 		r["fileName"].setValue( self.__largeFilePath )
 
-		for i in range( 20 ) :
+		for i in range( 5 ) :
```

Comment:
We can just remove the entire test instead - it was related to the OIIO ImageCache that we're no longer using. When removing it, could you include a reference to https://github.com/GafferHQ/gaffer/issues/1923, since that has a bunch of relevant history...